solutions/script-component-ad/components/AdBanner.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { CSSProperties, useEffect } from 'react'
+
+type Props = Pick<CSSProperties, 'height' | 'width'>
+
+export const AdBanner: React.FC<Props> = (props) => {
+  const [loading, setLoading] = React.useState(true)
+  useEffect(() => {
+    setTimeout(() => {
+      if (typeof window !== 'undefined') {
+        const { googletag } = window
```

Comment:
Is this always available with `afterInteractive` ?

Reply:
Tested without the delay and yep it works 👍